wp-parsely.php

```diff
@@ -168,10 +168,15 @@ function parsely_recommended_widget_register(): void {
  *
  * @since 2.6.0
  *
+ * @param Parsely|string|null $parsely The Parsely object to pass to the integrations.
  * @return Integrations
  */
-function parsely_integrations(): Integrations {
-	$parsely_integrations = new Integrations();
+function parsely_integrations( $parsely = null ): Integrations {
+	if ( empty( $parsely ) || get_class( $parsely ) !== Parsely::class ) {
```

Comment:
when would `$parsely` _not_ be null? this function is only called by the init action, correct?